SConstruct

```diff
@@ -1589,6 +1589,40 @@ for yaml in mglob(env, "data", "yaml"):
     dest = pjoin("build", "data", yaml.name)
     build(env.Command(dest, yaml.path, Copy("$TARGET", "$SOURCE")))
 
+# Copy Chemkin input files
+ck_sources = [
+    dict(input='data/inputs/gri30.inp',
+         thermo='data/thermo/gri30_thermo.dat',
+         transport='data/transport/gri30_tran.dat',
+         bibtex='data/bibtex/gri30.bib'),
+    dict(input='data/inputs/air.inp',
+         transport='data/transport/gri30_tran.dat'),
+    dict(input='data/inputs/argon.inp',
+         thermo='data/thermo/gri30_thermo.dat',
+         transport='data/transport/gri30_tran.dat'),
+    dict(input='data/inputs/airNASA9.inp',
+         thermo='data/thermo/airDataNASA9.dat',
+         bibtex='data/bibtex/airNASA9.bib'),
+    dict(input='data/inputs/h2o2.inp',
+         transport='data/transport/gri30_tran.dat'),
+    dict(input='data/inputs/silane.inp'),
+    dict(thermo='data/thermo/nasathermo.dat')
+]
+
+for mech in ck_sources:
+    if 'input' in mech:
+        build(env.Command('build/chemkin/{}'.format(mech['input'].split('/')[-1]),
+                          mech['input'], Copy('$TARGET', '$SOURCE')))
+    if 'thermo' in mech:
+        build(env.Command('build/chemkin/{}'.format(mech['thermo'].split('/')[-1]),
+                          mech['thermo'], Copy('$TARGET', '$SOURCE')))
+    if 'transport' in mech:
+        build(env.Command('build/chemkin/{}'.format(mech['transport'].split('/')[-1]),
+                          mech['transport'], Copy('$TARGET', '$SOURCE')))
+    if 'bibtex' in mech:
+        build(env.Command('build/chemkin/{}'.format(mech['bibtex'].split('/')[-1]),
+                          mech['bibtex'], Copy('$TARGET', '$SOURCE')))
```

Comment:
This can obviously be simplified, but I opted to leave the groupings of the `ck2yaml` inputs for the time being.